src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double RADIUS = 10.0;
+    private static final int MAX_VALUE = 20;
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(5);
+        switch (index) {
+            case 1:
+                return new Square(supplier.getRandomColor(),
+                        random.nextInt(MAX_VALUE));
+            case 2:
+                return new RightTriangle(supplier.getRandomColor(),
+                        random.nextInt(MAX_VALUE),random.nextInt(MAX_VALUE));
```

Comment:
let's be on the same page regarding the code style, same for all of the other similar situations
```suggestion
                        random.nextInt(MAX_VALUE), random.nextInt(MAX_VALUE));
```